resources/views/store/printables/families.blade.php

```diff
@@ -0,0 +1,66 @@
+@extends('store.layouts.printable_master')
+
+@section('title', $sheet_title)
+
+@section('content')
+
+  @include('store.printables.partials.masthead', ['specificPrintNote' => 'Ideally you should print this form every week to keep voucher allocations as up to date as possible.'])
+
+    <div class="content families">
+      <h1>Weekly Voucher Collection Sheet</h1>
+      <table class="info centre">
+        <tr>
+          <td>
+            <h2>Children's Centre: {{ $centre->name }}</h2>
+          </td>
+          <td class="week-commencing">
+            <p>Week commencing</p>
+            <img src="{{ asset('store/assets/date-field.svg') }}">
+          </td>
+        </tr>
+      </table>
+      <table class="families_table">
+        <tr>
+          <th>Main carer's name</th>
+          <th>RV-ID</th>
+          <th class="sml-cell">Voucher allocation</th>
+          <th class="sml-cell">Vouchers given out</th>
+          <th>Voucher numbers</th>
+          <th>Date collected</th>
+          <th class="lrg-cell">Signature</th>
+        </tr>
+        @foreach ($registrations as $registration)
+        <tr>
```

Comment:
Might want to wrap this whole block in if ($registration->family) to avoid unexpected 500s. They always catch me out in production.